runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -54,7 +57,7 @@
 	private final ConfigManager configManager;
 	private final ItemManager itemManager;
 	private final ClueScrollService clueScrollService;
-
+	private Item[] bankItems;
```

Comment:
Tag manager shouldn't be aware of the itemcontainer items.

Reply:
Relocated the itemcontainer items array to its own class.